### PR TITLE
Update Translations to use `xx-XX` syntax

### LINKED ISSUE #1336: Switch from fr.json translation file syntax to fr-FR.json syntax
We should have only `xx-XX.json` type of file here: https://github.com/busyorg/busy/tree/master/src/client/locales

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1336

Changes
- Remove all `xx.json` files
- Update Settings page to use `xx-XX` format

### CODE REVIEW

**Comment by bonustrack** [comment]
The browser language detection not seem to be working anymore. I use to see the website in french when unlogged.

File: src/client/settings/Settings.js [hunk 1]
```diff
@@ -101,46 +101,46 @@ export default class Settings extends React.Component {
...
-    hi: 'हिन्दी - Hindi',
-    as: 'অসমীয়া - Assamese',
-    bn: 'বাংলা - Bengali',
-    ta: 'தமிழ் - Tamil',
-    lo: 'ພາສາລາວ - Lao',
-    th: 'ภาษาไทย - Thai',
-    ko: '한국어 - Korean',
-    ja: '日本語 - Japanese',
-    zh: '简体中文 - Simplified Chinese',
+    'ar-SA': 'العربية - Arabic‏',
```
Comment (bonustrack) [inline src/client/settings/Settings.js]: Can you keep the list in the exact same order than before? With English first

Comment (jm90m) [reply src/client/settings/Settings.js]: Can we keep it alphabetized its easier to read and make changes to?

Comment (jm90m) [reply src/client/settings/Settings.js]: yeah for settings list, i see, we can since this displays dropdown

**Comment by bonustrack** [comment]
Make sure to match RTL language again on this PR @jm90m

**Comment by jm90m** [comment]
yeah will do

**Pull request merged**